python/ray/experimental/raysort/main.py

```diff
@@ -30,27 +35,47 @@ def get_args():
     )
     parser.add_argument(
         "--total_data_size",
-        default=1_000_000_000,
+        default=2 * 1000 * 1024 * 1024 * 1024,
         type=ByteCount,
-        help="partition size in bytes",
+        help="total data size in bytes",
     )
     parser.add_argument(
         "--num_mappers",
-        default=4,
+        default=512,
         type=int,
         help="number of map tasks",
     )
+    parser.add_argument(
+        "--num_mappers_per_round",
+        default=16,
+        type=int,
+        help="number of map tasks per first-stage merge tasks",
+    )
     parser.add_argument(
         "--num_reducers",
+        default=16,
+        type=int,
+        help="number of second-stage reduce tasks",
+    )
+    parser.add_argument(
+        "--num_concurrent_rounds",
         default=4,
         type=int,
-        help="number of reduce tasks",
+        help="max number of rounds of map/merge tasks in flight",
+    )
+    parser.add_argument(
```

Comment:
Correct me if I'm wrong, but it looks like this and reducer_output_chunk are not actually implemented yet, is that right? How about we remove these for now?

Reply:
Yea, it's only implemented in the C++ merge. I can remove these for now.

Reply:
Bump?